c#/Mandoline.Api.Examples/Client.Repl/Program.cs

```diff
@@ -4,54 +4,52 @@
 // root for full license information.
 // </copyright>
 
-namespace Client.Repl
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Core;
+using Sharprompt;
+
+namespace Client.Repl;
+
+internal class Program
 {
-    using System;
-    using System.Collections.Generic;
-    using System.ComponentModel.DataAnnotations;
-    using System.Linq;
-    using System.Text;
-    using System.Threading.Tasks;
-    using System.Xml.Linq;
-    using Core;
-    using Sharprompt;
-    internal class Program
+    private static async Task Main(string[] args)
     {
+        // Entry point fo REPL command line interface
+        Dictionary<string, Func<Task>> commands = new Dictionary<string, Func<Task>>();
```

Comment:
you can use var here on the left as it is clear what you are instantiating from the right-hand side